### ISSUE TITLE
As a developer I want the targetplatform file (n4tp) file to be removed

### ISSUE THREAD

**Issue by yoosiba** [eclipse/n4js#651]
Target platform files (`*.n4tp`) are used to define workspace wide dependencies, in particular:

1. used by the headless compiler to instal given set of dependencies priori to compilation of projects
2. used by the IDE to install set of dependencies to make them available to all projects in the workspace.

The `*.n4tp` files are obsolete, as all the information can be derived from `manifest.n4mf` files.
In fact that is already done in IDE with feature added in #512.
Once #654 lands on master, targetplatform file is obsolete from user point of view.

It is expected that targetplatform file will be removed from
* IDE - the maintenance actions in the Library manager
* N4JSC - the command line option for specifying targetplatfrom file

Optionally it has to be checked how targetplatform file is used internally and if it makes sense to remove it completely.
